doc/zmq.md

```diff
@@ -102,7 +104,17 @@ is structured as the following based on the type of message:
 
 Where the 8-byte uints correspond to the mempool sequence number.
 
-These options can also be provided in bitcoin.conf.
+`rawtx`: notifies when a transaction is added to mempool. The body of the message is the raw transaction in hexadecimal format.
+
+`hashtx`: notifies when a transaction is added to mempool. The body of the message is structured in the following way:
+
+    hashtx <32-byte transaction hash>
+
```

Comment:
Both `rawtx` and `hashtx` notify about all transactions. Mempool and transactions in blocks. This means it could happen that a transaction is published multiple times. First when it enters the mempool and then again in each block that includes it. This should be documented.

Reply:
Done.